lib/actions/api.js

```diff
@@ -91,6 +88,24 @@ function getActiveItinerary(state) {
   return activeItinerary
 }
 
+/**
+ * Dispatches a method from either v1actions or v2actions, depending on
+ * which version of OTP is specified in the config.
+ * @param {*} methodName    the method to execute
+ * @param  {...any} params  varargs of params to send to the action
+ */
+function executeOTPVersionSpecificAction(methodName, ...params) {
```

Comment:
Pretty cool! Maybe you can rename to just `executeOTPAction`.

Reply:
We sure can! Good suggestion thank you

Reply:
Renamed in a500cbe779f68c61c9e9739c30b640641fdb2b1a